src/axom/inlet/Field.hpp

```diff
@@ -257,6 +258,17 @@ class Field : public std::enable_shared_from_this<Field>
    *****************************************************************************
   */
   std::shared_ptr<Field> validValues(const std::initializer_list<double>& set);
+
+  std::shared_ptr<Field> registerVerifier(std::function<bool()> lambda);
+  bool verify();
```

Comment:
If i read this correctly this function (verify) is not meant to be called by the user but is called by the Inlet::verify function.  It needs to be documented and that explicitly called out.